.github/workflows/actions.yml

```diff
@@ -0,0 +1,19 @@
+name: actions
+on:
+  push:
+    branches: [ github-actions ]
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Permissions for tabularasa.sh
+        run: chmod +x ./tabularasa.sh
+      - name: Workaround for yarn integrity issue
```

Comment:
Did you try the workflow without this step? The script is only meant for us to use when we really need to start from scratch (Tabula Rasa = "blank slate") and ideally, we don't have to do this every time we start a new CI action.

We should remove this (and the previous step to set permissions) if it isn't absolutely necessary.